src/status_im2/subs/root.cljs

```diff
@@ -111,6 +111,7 @@
 (reg-root-key-sub :chats/mention-suggestions :chats/mention-suggestions)
 (reg-root-key-sub :chat/inputs-with-mentions :chat/inputs-with-mentions)
 (reg-root-key-sub :chats-home-list :chats-home-list)
+(reg-root-key-sub :shared-element-id :shared-element-id)
```

Comment:
could you please add namespace to the keyword, with context name

Reply:
Maybe I can move it to the top under the `;;view` part

Reply:
yes or in general